corehq/apps/locations/adjacencylist.py

```diff
@@ -222,8 +224,24 @@ def get_queryset_descendants(self, queryset, include_self=False):
             cte_set = None
         return ComparedQuerySet(mptt_set, cte_set, timing)
 
+    @contextmanager
+    def delay_mptt_updates(self):
+        yield
```

Comment:
Just realized this might be able to be removed since the refactor of bulk management.